tests/FrameCameraTests.cpp

```diff
@@ -94,6 +163,57 @@ class FrameSensorModel : public ::testing::Test {
       }
 };
 
+
+
+INSTANTIATE_TEST_CASE_P(JacobianTest,FramerParameterizedTest,
```

Comment:
I think we want to change the first argument for this to something more general. Right now the test name will be
`JacobianTest/FramerParameterizedTest.JacobianTest/#`. Changing it to something like `FramingSensorModel` would give us a test called `FramingSensorModel/FramerParameterizedTest.JacobianTest/#`.